source-file-interceptor/source-file-interceptor.php

```diff
@@ -151,7 +156,7 @@ function inject_source_ctx($parsed_args, $url) {
 
 
 
-	return $parsed_args;
+	return $response;
 }
 
-add_filter('http_request_args', 'inject_source_ctx', 999, 3);
+add_filter('http_response', 'inject_source_ctx', 999, 3);
```

Comment:
Does this mean that we're not logging the request anymore?

Reply:
The `http_request_args` hook returns the `parsed_args` and `url`.
The new hook `http_response` returns `response` , `parsed_args` and `url`. 

So it is logging the same as before with an additional parameter 

https://developer.wordpress.org/reference/hooks/http_request_args/

Reply:
Ahh, nice one. Thanks